Modules/BarcodeScanner/BarcodeType.swift

```diff
@@ -0,0 +1,122 @@
+import AVFoundation
+
+/// An enumeration describing barcode types which are supported by the `BarcodeScannerView`.
+public enum BarcodeType {
```

Comment:
This micro framework is wrapping the complete AVMetadata implementation of Apple. So it shouldn't expose Apple API. This will guarantee API stability even if you exchange the underlying implementation.